src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form-data.ts

```diff
@@ -9,6 +9,12 @@ export class PoolFormData {
   crushInfo = false;
   applications: any;
 
+  readonly APP_LABELS: Record<string, string> = {
+    cephfs: 'Filesystem',
+    rbd: 'Block',
+    rgw: 'Object'
+  };
+
```

Comment:
@aaSharma14 instead of hard-coding this info here, why not checking with the RADOS team (CC: @rzarzynski @ljflores @yaarith) to include that metadata in the core? Many of the Dashboard improvements might also make sense from CLI UX perspective (the frequently overlooked 😮‍💨), and it'd be easier to maintain if they "emerge" from the core.

Another example, we (Dashboard team) have been recently talking about the need to add multiple application tags (e.g: `rbd` + `nvmeof`, or `cephfs` + `nfs`) to indicate Ceph-native and standard protocols.

https://github.com/ceph/ceph/blob/a12f5b55472090967ffe9af78f3fca338a57860d/src/osd/osd_types.cc#L1562-L1564

Reply:
This can be an improvement after T, @aaSharma14 lets create a tracker for this and discuss in it next syncup. Thanks @epuertat